stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/options/MessageOptionsView.kt

```diff
@@ -27,34 +28,82 @@ public class MessageOptionsView : FrameLayout {
         defStyleAttr
     )
 
-    internal fun configure(configuration: Configuration, isMessageTheirs: Boolean) {
+    internal fun configure(configuration: Configuration, isMessageTheirs: Boolean, syncStatus: SyncStatus) {
+        if (isMessageTheirs) {
+            configureTheirsMessage(configuration, syncStatus)
+        } else {
+            configureMineMessage(configuration, syncStatus)
+        }
+    }
+
+    private fun configureTheirsMessage(configuration: Configuration, syncStatus: SyncStatus) {
         val iconsTint = configuration.iconsTint
 
-        binding.replyTV.configureListItem(configuration.replyText, configuration.replyIcon, iconsTint)
-        binding.threadReplyTV.configureListItem(configuration.threadReplyText, configuration.threadReplyIcon, iconsTint)
+        // Todo: Uncomment once reply is done.
+        // binding.replyTV.configureListItem(configuration.replyText, configuration.replyIcon, iconsTint)
 
-        if (configuration.copyTextEnabled) {
-            binding.copyTV.isVisible = true
-            binding.copyTV.configureListItem(configuration.copyText, configuration.copyIcon, iconsTint)
-        } else {
-            binding.copyTV.isVisible = false
+        if (syncStatus == SyncStatus.COMPLETED) {
```

Comment:
Can _their_ message be in any another sync status somehow?

Reply:
Indeed, COMPLETED is the only possible way.